server/common/ResponseBuilder.java

```diff
@@ -143,34 +310,33 @@ public static StatusRuntimeException exception(Throwable e) {
 
     public static class Concept {
 
-        public static ConceptProto.Concept concept(grakn.core.concept.Concept concept) {
+        public static ConceptProto.Concept protoConcept(grakn.core.concept.Concept concept) {
```

Comment:
Why are these renamed to `protoConcept`, etc? The `proto` prefix seems redundant given this file's whole purpose is to create proto objects.

Reply:
We're doing this because this method is quite exceptional in that the input type, return type, and method name, are all the same word "concept" - it got kind ambiguous.